packages/calypso-e2e/src/lib/components/editor-sidebar-block-inserter-component.ts

```diff
@@ -4,7 +4,8 @@ import { EditorComponent } from './editor-component';
 
 const sidebarParentSelector = '.block-editor-inserter__main-area';
 const selectors = {
-	closeBlockInserterButton: 'button[aria-label="Close block inserter"]',
+	closeBlockInserterButton:
+		'button[aria-label="Close Block Inserter"], button[aria-label="Close block inserter"]',
```

Comment:
I'm assuming once GB 19.5.0 is released, we won't need the old selector?

Reply:
I'm not sure as well. Maybe it's worth adding a comment?